Dockerfile.django

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:20.04
+
+RUN apt-get update && apt-get install -y python3 pipenv python3-pip
+
+WORKDIR /organizer
+COPY . /organizer
+
+RUN pipenv lock -r > requirements.txt
```

Comment:
Why pip2 instead of pipenv?

Reply:
Since this is in a container, using a virtualenv is superfluous so I only use pipenv to check all of the dependencies and generate the `requirements.txt`. Also, oops, I meant to use `pip3` here.